tests/lint-json.php

```
@@ -12,41 +12,256 @@
+        new RecursiveCallbackFilterIterator(
+            new RecursiveDirectoryIterator($root, FilesystemIterator::SKIP_DOTS),
+            static function (SplFileInfo $file, string $_key, RecursiveDirectoryIterator $iterator) use ($root): bool {
+                if ($iterator->hasChildren()) {
+                    if (in_array($file->getFilename(), PRUNED_DIRECTORY_NAMES, true)) {
+                        return false;
+                    }
+
+                    $depth = substr_count($file->getPathname(), '/') - substr_count($root, '/');
+
+                    if ($depth >= MAX_SCAN_DEPTH) {
+                        throw new UnexpectedValueException(sprintf(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '164', column '35'). (undefined)

(MissingImport)